sdk/monitor/opentelemetry-exporter/samples/@azure/storage-blob/javascript/tracing.js

```diff
@@ -0,0 +1,22 @@
+import { AzureMonitorTraceExporter } from "@azure/monitor-opentelemetry-exporter";
```

Comment:
If this file + importing it is the only real novel change, I don't think it's really worth copying each of the storage-blob samples.

I feel like just doing one sample with the automatic provider would be enough and perhaps a second example that used manual span propagation would be interesting?

Reply:
Removed the rest and added an "optional" snippet of creating a root span.